superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -261,7 +261,7 @@ export default class CRUDCollection extends React.PureComponent<
             aria-label="Delete item"
             className="pointer"
             data-test="crud-delete-icon"
-            iconSize="m"
+            style={{fontSize: '18px'}}
```

Comment:
```suggestion
            style={{fontSize: '18px' }}
```

Reply:
can we change this to use the css prop instead of style? That's a pattern we've been trying to follow. Thanks!